config/routes.rb

```diff
@@ -1,4 +1,17 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  devise_for :users, path: '', path_names: {
+    sign_in: 'login',
+    sign_out: 'logout',
+    registration: 'signup'
+  }, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+  resources :reservations
+  resources :cars
+  resources :users
```

Comment:
- [ ] As the previous reviewer commented, I recommend filtering your routes with `except` and `only` so you actually get only the necessary routes. 👍

Reply:
Consider this done and some other adjustments done too